content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -1317,6 +1330,49 @@ created Pod should be ready without any of its containers crashing, for it to be
 This defaults to 0 (the Pod will be considered available as soon as it is ready). To learn more about when
 a Pod is considered ready, see [Container Probes](/docs/concepts/workloads/pods/pod-lifecycle/#container-probes).
 
+### Pod Replacement Policy
+
+{{< feature-state feature_gate_name="DeploymentPodReplacementPolicy" >}}
+
+{{< note >}}
+This feature is disabled by default. You can enable it by setting the `DeploymentPodReplacementPolicy`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/)
+and on the [Kube Controller Manager](/docs/reference/command-line-tools-reference/kube-controller-manager/)
+{{< /note >}}
+
+Deployment Pods can become terminating due to deletion, usually during the rollout or scaling of a new
+deployment. Some pods may take a long time to terminate and may consume additional resources during this
```

Comment:
```suggestion
Pods that (indirectly) belong to a Deployment can become terminating due to deletion, usually during
the rollout of a change or an update to the scale of an existing Deployment.
After a change to a Deployment, some pods may take a long time to terminate and may consume
additional resources during that period of
```

Reply:
- I think using `(indirectly)` is a bit confusing. The relationship is quite direct via the owner references.
- `rollout of a change` is also a bit confusing as it is possible to do a rollout without any change: `kubectl rollout restart`